django_structlog_demo_project/taskapp/celery.py

```diff
@@ -102,6 +102,9 @@ def rejected_task():
     def unknown_task():
         """Simulate a task unavailable in the worker for demonstration purpose"""
 
+else:  # pragma: no cover
+    pass
```

Comment:
FYI there’s `pragma: no branch` that can be used on the if line, to avoid the need for an empty else clause
https://coverage.readthedocs.io/en/7.3.2/branch.html#structurally-partial-branches

Reply:
thanks, ill change it.